src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -374,6 +374,16 @@ export const isUploading781Form = formData =>
 export const isUploading781aForm = formData =>
   _.get('view:upload781aChoice', formData, '') === 'upload';
 
+export const isAnswering781Questions = formData =>
+  _.get('view:uploadPtsdChoice', formData, '') === 'answerQuestions' &&
+  _.get('view:doneEnteringIncidents', formData, false) !== true &&
+  needsToEnter781(formData);
+
+export const isAnswering781aQuestions = formData =>
+  _.get('view:uploadPtsdChoice', formData, '') === 'answerQuestions' &&
+  _.get('view:doneEnteringSecondaryIncidents', formData, false) !== true &&
```

Comment:
maybe assign a default value to `view:doneEnteringSecondaryIncidents` so "equals false" can be used.